### DISCUSSION TITLE
NetBeans failure

### DISCUSSION THREAD

**Discussion by gauravsubedi191** [apache/netbeans#3485] [Q&A]
hello! I have problem running netbeans the full x64 version in my laptop. It simply wouldn't run from the shortcut created during installation. I need to go to program files  and find the another icon from which it runs and it lacks many feature. So i wonder if anyone can help me with it.

**Comment by asbachb** [comment]
Try to run from command line which might show some errors and give us some more cookies like your os, used java versions, etc.

Without that information nobody is able to help.

**Reply by oyarzun** [reply]
@chatttan23 @asbachb  I wonder if it is the same as this issue. https://issues.apache.org/jira/browse/NETBEANS-6399

**Reply by gauravsubedi191** [reply]
it is somewhat same. 
![image](https://user-images.githubusercontent.com/83532694/151174620-b2d5f490-1dba-4270-a8db-9470a65d03a9.png)
see here the two application extension is available. The one with netbeans64 should be the one full version to be running. But it doesn't run. The upper one only runs but lacks many things compared to another.

**Reply by neilcsmith-net** [reply]
@chatttan23 can you share the top section from `View / IDE Log` with the version that does run?

**Reply by oyarzun** [reply]
@chatttan23 the log would useful. Also what JDK are you using? Is it a 64bit JDK?

**Reply by gauravsubedi191** [reply]
yes i am. and i really need netbeans but its not opening. How can i show any log if its not running??

**Reply by neilcsmith-net** [reply]
@chatttan23 you said it was running with some elements missing!

**Comment by neilcsmith-net** [comment]
@chatttan23 this is Windows? It sounds like something possibly went awry during installation.  We (as in my company) also have an unofficial installer built with InnoSetup that packages the official zip release which you might want to try? https://www.codelerity.com/netbeans/

**Reply by gauravsubedi191** [reply]
i Tried installing it too but the problem is still not solved.
I even cleared Netbeans cache and everything to do it properly like a first time but it does not solve the problem.

**Reply by neilcsmith-net** [reply]
@chatttan23 Are you using the shortcut or directly executing the files in `bin/`? You have to use the created shortcuts with our installer at the moment - they pass an essential argument to the launcher.

**Reply by gauravsubedi191** [reply]
i tried to use both. but not one of them was working.

**Reply by neilcsmith-net** [reply]
Both what?  If you're trying our installer, which includes its own JDK, then use the desktop / start menu shortcut and make sure other version of NetBeans are not installed.  Neither of the `.exe` files in `bin` will work for you.

If you're using the Apache provided installer instead of the Codelerity one, you need a JDK installed. What JDK are you using? What is the JDK written in to the `etc/netbeans.conf` file by the Apache installer?  Is it the right one?

**Comment by geertjanw** [comment]
"The upper one only runs but lacks many things compared to another."

What does this mean? Can you be very specific, so we can help you?

**Reply by gauravsubedi191** [reply]
![image](https://user-images.githubusercontent.com/83532694/152546224-5e1be77d-1b7e-4cb3-9dac-6a094166eee9.png)
So, there are two application files for x64 and x32 . The one that should be running is x64 instead x32 runs. Which lacks some plugins compared to x64 and can cause problems in coding.

**Reply by gauravsubedi191** [reply]
Although i already unistalled it and reinstalled quite a few times now. but nothing seems to solve my problem.

**Comment by geertjanw** [comment]
"The upper one only runs but lacks many things compared to another."

What does this mean? Can you be very specific, so we can help you?
